.github/workflows/actions.yml

```diff
@@ -17,12 +17,12 @@ jobs:
       fail-fast: false
       matrix:
         backend: [tensorflow, jax, torch]
-        version: [stable]
+        version: [keras-stable]
         include:
           - backend: jax
-            version: 3.1
+            version: keras-3.1
```

Comment:
Doesn't this need to be the latest Keras release?

Reply:
This is a test we added recently to try running against Keras a few versions back. Just makes sure we are staying compatible.

Reply:
Oh, that's great!